code/orca/farm/main.en.ts

```diff
@@ -0,0 +1,30 @@
+import { getOrca, OrcaFarmConfig, OrcaPoolConfig } from "@orca-so/sdk";
+import { Connection, Keypair } from "@solana/web3.js";
+import { readFile } from "mz/fs";
+import Decimal from "decimal.js";
+
+(async () => {
+  const secretKeyString = await readFile(
+    "/Users/scuba/my-wallet/my-keypair.json",
+    {
+      encoding: "utf8",
+    }
+  );
+  const secretKey = Uint8Array.from(JSON.parse(secretKeyString));
+  const owner = Keypair.fromSecretKey(secretKey);
```

Comment:
Could we do the same pattern we use everywhere else instead?

IE, generate keypair and airdrop SOL to it? Just make this hit devnet instead of mainnet-beta